### PR TITLE
WS1 - Static Recipe - diego97jaramillo

### PR DESCRIPTION
**Pull request by diego97jaramillo**
### Acceptance Criteria

- [x] You must use good [semantics in HTML tags](https://es.semrush.com/blog/html-semantico/).
- [x] You have to import fonts from a CDN or create a font from 0 with [@font-face](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face).
- [x] Use [SASS preprocessor](https://sass-lang.com/documentation/style-rules/) or others ([Less vs Sass vs Stylus](https://stackshare.io/stackups/less-vs-sass-vs-stylus)).
- [x] Use [variables](https://sass-lang.com/documentation/variables/) for style sheets without hard code.
- [x] You have to [reset the CSS](https://www.aluracursos.com/blog/reset-css-que-es-ejemplos-como-crear-y-utilizar) so that you can work with the HTML elements in a clean way.
- [x] You have to use properties of [box model](https://web.dev/i18n/es/learn/css/box-model/) such as box-sizing, margin, padding, and border.
- [x] You have to use CSS display layout properties:
  - [CSS Inline vs Inline-Block vs Block](https://www.samanthaming.com/pictorials/css-inline-vs-inlineblock-vs-block/)
  - [Grid & flexbox MDN](https://developer.mozilla.org/es/docs/Web/CSS/CSS_grid_layout/Relationship_of_grid_layout_with_other_layout_methods#grid_y_flexbox)
- [x] You have to use CSS [positioning](https://css-tricks.com/almanac/properties/p/position/) and [z-index](https://css-tricks.com/css-painting-order/) properties to place the overlapping elements.

### Demo
imagen

---
### Comentarios Taller:
- Falta **1** criterio de aceptación.
**Nota:** `5.0 de 5.0`
> Buen trabajo!

### CODE REVIEW

**Comment by DanteCely** [comment]
Te falta implementar SASS 🙄

File: diego97jaramillo/1-cooking-recipe-html-css/Cooking-recipe.html
Comment (diego97jaramillo) [inline diego97jaramillo/1-cooking-recipe-html-css/Cooking-recipe.html:1]: @DanteCely marvin ya quedó el final de mi receta

File: diego97jaramillo/cooking-book/pages/cooking-recipe/Cooking-recipe.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                            </div>
+                            <div>
+                                <i class="fa-solid fa-euro-sign"></i>
+                                <span>Bajo</span>
+                            </div>
+                            <div>
+                                <i class="fa-solid fa-hat-chef"></i>
+                                <span>Facil</span>
+                            </div>
+                        </div>
```
Comment (DanteCely) [inline diego97jaramillo/cooking-book/pages/cooking-recipe/Cooking-recipe.html:55]: Veo que puedes cambiar los div con más semántica, pueden ser una lista no ordenada:
```
                        <ul class="steps-icons">
                            <li>
                                <i class="fa-solid fa-person"></i>
                                <span>4</span>
                            </li>
                           ...
```

File: diego97jaramillo/cooking-book/pages/cooking-recipe/Cooking-recipe.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                        <article>
+                            <h3>Paso a Paso</h3>
+                            <p>Es una olla con abundante agua hirviendo, ponemos la sal y los espaguetis. No hay que poner aceite ni nada, solo abundante agua con sal.</p>
+                            <p>Los hacemos 1 minuto menos de lo que diga el fabricante, para que queden al dente. Hay que tener en cuenta que cuando los saquemos todavia se seguiran haciendo.</p>
+                            <p>En un bol batimos los dos huevos y las dos yemas.</p>
+                            <p>Le añadimos el queso parmesano y mezclamos bien.</p>
+                            <p>En una sartén hacemos el bacon que hemos cortado en tiras.</p>
+                            <p>Cuando los espaguetis estén listos, los llevamos directamente de la olla al bol con el huevo. Revolvemos bien.</p>
+                            <p>Añadimos el bacon de la sartén y una cucharada de pimienta y volvemos a mezclar muy bien.</p>
+                            <p>Servimos inmediatamente.</p>
```
Comment (DanteCely) [inline diego97jaramillo/cooking-book/pages/cooking-recipe/Cooking-recipe.html:74]: El paso a paso puede ser una lista ordenada ya que es un paso a paso.
```
<ol>
  <li>
      Es una olla con abundante agua hirviendo, ponemos la sal y los espaguetis. No hay que poner aceite ni nada, solo abundante agua con sal.
  </li>
  ...
```

File: diego97jaramillo/1-cooking-recipe-html-css/Cooking-recipe.html
Comment (DanteCely) [reply diego97jaramillo/1-cooking-recipe-html-css/Cooking-recipe.html:1]: Listo ahí la reviso

**Review by DanteCely — APPROVED** [review]

**Pull request merged**